src/vmm/src/vmm_config/machine_config.rs

```diff
@@ -84,6 +84,14 @@ impl HugePageConfig {
     pub fn is_hugetlbfs(&self) -> bool {
         matches!(self, HugePageConfig::Hugetlbfs2M)
     }
+
+    /// Gets the page size in KiB of this [`HugePageConfig`].
+    pub fn page_size_kib(&self) -> usize {
```

Comment:
codecov points at it not having a test

Reply:
True, its because its only callsite is not covered by unit tests :(
I can't really think of a meaningful unittest to cover this function that wouldn't just copy-paste what the function does. Do you have any ideas?

Reply:
Agree that such test won't be overly meaningful, but at least having it will spare us from fine-grained judgment calls in the future on whether some code deserves a test. Feel free to disagree if you have a solution to that.